features/test-project.feature

```diff
@@ -0,0 +1,152 @@
+Feature: Test Mocha Project
+  
+  Scenario: Testing a project
+    Given I visit sample project file "sample-project/test/parse-float-test.js"
+    When I switch to buffer "parse-float-test.js"
+    Then I should be in buffer "parse-float-test.js"
+    When I run the command "mocha-test-project"
+    And I wait for the compilation to finish
+    Then I should see buffer "*mocha tests*"
+    And I should see contents in buffer "*mocha tests*":
+      """
+      Mocha started
+      """
+      """
+      2 passing
+      """
+      """
+      1 failing
+      """
+      """
+      Mocha exited abnormally
+      """
```

Comment:
Yeah this totally doesn't work. I was previously calling the step multiple times, from a style perspective do you think that is better or using a table step?

I kind of lean towards a table step myself.